doc/changelogs.md

```diff
@@ -151,3 +151,35 @@ will be most convenient for our users.
 [gfm]: https://guides.github.com/features/mastering-markdown/
 [keep_a_changelog]: https://keepachangelog.com/en/1.0.0/
 [gitlab_changelog]: https://docs.gitlab.com/ee/development/changelog.html
+
+## Alternatives considered
+
+Prior to stick with current process we considered several alternatives. In
+short:
+
+* Collect titles of commits.
+* Collect titles of closed issues.
+* Write changelog entries in a cover letter / pull request and manually move to
+  a release notes draft on GitHub.
+* Single file changelog.
+
+See the previous section for reasons why we didn't pick up something from this
+list.
+
+More alternatives are below.
+
+### Store a changelog entry in a commit message
+
+This process is [used in librepo][librepo_changelog_process]. It should mostly
+work for us, but there are several downsides:
+
+* A developer should remember about the changelog when writting a commit
+  message.
+* There is no ability to modify a changelog entry if a mistake is found later.
+
+It is easy to forget to fix a mistake if we can do it only just before a
```

Comment:
I'd change wording: "... if we can only do it ...".

Reply:
I rewrote it entirely.